src/app/register/pages/register/register.component.html

```diff
@@ -0,0 +1,78 @@
+<main class="container ">
+  <div class="row space-around">
+    <div class="col l6 m8 s4" [ngClass]="{ 'no-gutters': platform.columns4 }">
+      <!-- MOBILE PRESENTATION -->
+      <mat-vertical-stepper
+        [linear]="isLinear"
+        #stepper
+        *ngIf="platform.columns8 || platform.columns4"
+        (selectionChange)="selectionChange($event)"
+        class="orcid-stepper-wizard"
+      >
+        <mat-step [stepControl]="FormGroupStepA">
+          <ng-template
+            matStepLabel
+            i18n="@@register.personalData"
+            i18n="@@register.personalData"
+            >Personal data</ng-template
+          >
+          <app-step-a [(formGroup)]="FormGroupStepA"></app-step-a>
+        </mat-step>
+        <mat-step [stepControl]="FormGroupStepB">
+          <ng-template matStepLabel i18n="@@register.personalSecurityVisibility"
+            >Security and visibility</ng-template
+          >
+          <app-step-b
+            [(formGroup)]="FormGroupStepB"
+            [personalData]="personalData"
+          ></app-step-b>
+        </mat-step>
+        <mat-step [stepControl]="FormGroupStepC" #lastStep>
+          <ng-template matStepLabel i18n="@@register.personalNotificationsTerm"
+            >Notifications and terms</ng-template
+          >
+          <app-step-c
+            [(formGroup)]="FormGroupStepC"
+            (submit)="register('vertical')"
+          ></app-step-c>
+        </mat-step>
+      </mat-vertical-stepper>
+
+      <!-- DESKTOP PRESENTATION -->
+      <mat-horizontal-stepper
+        [linear]="isLinear"
+        #stepper
+        *ngIf="platform.columns12"
+        labelPosition="bottom"
+        class="orcid-stepper-wizard"
+        (selectionChange)="selectionChange($event)"
+      >
+        <mat-step [stepControl]="FormGroupStepA">
+          <ng-template matStepLabel i18n="@@register.personalData"
+            >Personal data</ng-template
+          >
+          <app-step-a [(formGroup)]="FormGroupStepA"></app-step-a>
+        </mat-step>
+        <mat-step [stepControl]="FormGroupStepB">
+          <ng-template matStepLabel i18n="@@register.personalSecurityVisibility"
+            >Security and visibility</ng-template
+          >
+          <app-step-b
+            [(formGroup)]="FormGroupStepB"
+            [personalData]="personalData"
+            (submit)="submitStepB()"
```

Comment:
@leomendoza123 I think you should remove this since inside the component there is a button with matStepperNext

Reply:
👍